modules/sso/install

```diff
@@ -4,7 +4,7 @@ set -e
 
 SOURCES_DIR=/tmp/artifacts
 
-unzip -o $SOURCES_DIR/keycloak-server-overlay.zip -d $JBOSS_HOME
+unzip -o $SOURCES_DIR/keycloak-server-overlay*.zip -d $JBOSS_HOME
```

Comment:
@slaskawi Can you remove that asterisk from there yet? If there would be more than one _keycloak-server-overlay_ versions under that location, it would try to unzip them all. We want to be sure, which of them (the only one) is expanded & used.